requirements.txt

```diff
@@ -1,12 +1,23 @@
 black==23.3.0
+bpy-addon-build==0.1.7
```

Comment:
Any reason to not just specify this to be the latest? In general worried about running into "dependency hell" will so many dependencies, do we really need all of these below listed? Can't bpy-addon-build take care of its own requirements?

Reply:
I automatically generated the requirements.txt from my virtual environment. I'll be updating this later this week

Reply:
Can't it just use the requirements generated from within bpy-addon-build? And then the MCprep requirement would just be to pip install `bpy-addon-build` itself (latest, ideally) which would then account for its own dependencies. It would also help clarify we really only have *one* dependency, that itself just brings along with it a few other things.